mappings/net/minecraft/item/Item.mapping

```diff
@@ -18,11 +18,14 @@ CLASS net/minecraft/class_1792 net/minecraft/item/Item
 	METHOD method_19263 isFood ()Z
 		COMMENT Checks if this item is food and therefore is edible.
 	METHOD method_19264 getFoodComponent ()Lnet/minecraft/class_4174;
+		COMMENT @return this item's {@link FoodComponent}, or null if one was not set.
```

Comment:
```suggestion
		COMMENT Returns this item's {@link FoodComponent}, or {@code null} if one was not set.
```
Like in google java style, we don't use return tag if the return info is the only statement in the method description. When we are using java 15 to build, we can use `{@return}` inline tag at that time.

Reply:
Do note that Yarn branches for newer versions of Minecraft are built using Java 16, so inline return tags can be used. This pull request should be updated accordingly.

For more information, see #2454.